changelog/changelog.go

```diff
@@ -131,6 +132,16 @@ func ParseOne(reader *bufio.Reader) (*ChangelogEntry, error) {
 	return &changeLog, nil
 }
 
+func ParseFileOne(path string) (*ChangelogEntry, error) {
```

Comment:
Not a huge fan of this name, but wasn't sure what else to call it. :disappointed: